messaging/WFLY-14087_documentation_connection_pool_configuration.adoc

```diff
@@ -0,0 +1,62 @@
+= Document how to configure a datasource to take advantage of new Artemis pool support
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          jms,messaging
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14087[WFLY-14087]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-13226[WFLY-13226]
+* https://issues.apache.org/jira/browse/ARTEMIS-2823[ARTEMIS-2823]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [X] QE
+
+=== Affected Projects or Components
+
+* WildFly
+* Apache Artemis
+
+=== Other Interested Projects
+
+== Requirements
+
+With Apache Artemis 2.16, proper support for a JDBC pool was introduced. While apache Artemis uses https://commons.apache.org/proper/commons-dbcp/configuration.html[Apache Commons DBCP], it allows a better support for other pools too.
+This change requires to provide proper configuration of the Ironjacamar Datasource for good perfomance. Thus we need to document this in our community doc.
+
+=== Hard Requirements
+
+N/A
+
+=== Nice-to-Have Requirements
+
+N/A
+
+=== Non-Requirements
+
+N/A
+
+== Test Plan
+
```

Comment:
What is necessary to test from the dev's point of view? Full test plan will be prepared separately by QE, but this section in the analysis should contains key points that was find out during analysis preparation.

Reply:
TP section in analysis is still empty